spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/PoolControllerHelperService.java

```diff
@@ -808,15 +658,15 @@ public JSONObject createElementSelectDataTable(HttpSession session, JSONObject j
         for (LibraryDilution libraryDilution : requestManager.listAllLibraryDilutionsByPlatform((PlatformType.get(platform)))) {
           JSONArray pout = new JSONArray();
           pout.add(libraryDilution.getName());
-          pout.add(libraryDilution.getConcentration());
+          pout.add(String.format("%.1f", libraryDilution.getConcentration()));
```

Comment:
Is there a reason to limit to 1 decimal place? If so, should input be validated similarly?